canvas_site_creator/utils.py

```diff
@@ -47,9 +47,12 @@ def create_canvas_course_and_section(data):
     template_id = data['template_id']
 
     # The course may be associated with either a department or coursegroup (depending on the school)
-    department_or_course_group_id = course.department_id if course.department_id else course.course_group_id
+    if course.department_id:
```

Comment:
I think we might want to reverse this, and explicitly limit to `colgsas`. E.g. 
```
if course.school_id == 'colgsas' and course.course_group_id:
    # use the course group id
else:
    # use the department id
```
In the past we've erroneously received course group IDs in the DCE feed, and many/all FAS courses will have both a department and a course_group. For FAS courses we want to prefer the course_group, and for all other schools we want to ignore course_groups altogether.

Reply:
Updated in [d1d5926](https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/pull/408/commits/d1d592648b1957087e7507a4346f9ff805de8487).